language/de/acp/board.php

```diff
@@ -399,6 +401,8 @@
 	'LOAD_CPF_VIEWTOPIC'			=> 'Benutzerdefinierte Profil-Felder in der Themen-Ansicht anzeigen',
 	'LOAD_USER_ACTIVITY'			=> 'Aktivität der Mitglieder anzeigen',
 	'LOAD_USER_ACTIVITY_EXPLAIN'	=> 'Zeigt im Profil und im persönlichen Bereich an, in welchen Foren und Themen ein Mitglied am aktivsten ist. Es wird empfohlen, diese Funktion in Foren zu deaktivieren, die mehr als eine Million Beiträge haben.',
+	'LOAD_USER_ACTIVITY_LIMIT'		=> 'Beitragsgrenze für Benutzeraktivität',
+	'LOAD_USER_ACTIVITY_LIMIT_EXPLAIN'	=> 'Bei Benutzern mit mehr Beiträgen als hier angegeben, wird das aktivste Thema/Forum nicht angezeigt. Es wird keine Beitragsgrenze festgelegt, wenn 0 als Wert eingestellt wird.',
```

Comment:
Vorschlag: Bei Benutzern, die mehr Beiträge haben als hier angegeben, wird nicht mehr angezeigt, in welchen Foren und Themen sie am aktivsten sind. Mit 0 werden diese Informationen bei allen Benutzern angezeigt.